### ISSUE TITLE
reward in PHC

### ISSUE THREAD

**Issue by interestingzhuo** [ZhengyiLuo/UniversalHumanoidControl#21]
![image](https://github.com/ZhengyiLuo/UniversalHumanoidControl/assets/43921151/379d043f-1e0f-43d4-a635-ed1eb9f7eede)
how to compute the energy penalty. I can only "get_actor_dof_forces" and "_rigid_body_ang_vel" by issaacgym.

**Comment by interestingzhuo** [comment]
Is the reward computed via dof(axis angle). I add the code in ASE code, but  the word becomes negative. I wonder is it  right?
```
dof_force_tensor = self.gym.acquire_dof_force_tensor(self.sim)
self.gym.refresh_dof_force_tensor(self.sim)
self._dof_force = gymtorch.wrap_tensor(dof_force_tensor).view(self.num_envs, dofs_per_env)    
def get_energy(self):
        dof_force, dof_vel, dof_obs_size, dof_offsets = self._dof_force, self._dof_vel, self._dof_obs_size, self._dof_offsets
        num_joints = len(dof_offsets) - 1
        num_env = dof_force.shape[0]
        energy = torch.zeros(num_env, device = dof_force.device)

        for j in range(num_joints):
            dof_offset = dof_offsets[j]
            dof_size = dof_offsets[j + 1] - dof_offsets[j]
            force = dof_force[:, dof_offset:(dof_offset + dof_size)]
            vel = dof_vel[:, dof_offset:(dof_offset + dof_size)]
            energy += torch.mul(force, vel).sum(dim = -1)**2
        energy =  0.00005*energy
        return energy

rew_buf += -1 * self.energy_buf
```
